processing/src/main/java/io/druid/segment/filter/ExtractionFilter.java

```diff
@@ -66,6 +68,10 @@ public ExtractionFilter(
   @Override
   public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
   {
+    final List<Filter> filters = makeFilters(selector);
+    if (filters.isEmpty()) {
+      return new WrappedConciseBitmap();
+    }
     return new OrFilter(makeFilters(selector)).getBitmapIndex(selector);
```

Comment:
it might make sense to fix OrFilter to take empty list instead, if that does not change the behavior.